manifest/manifest_iterator.cc

```diff
@@ -79,7 +79,7 @@ absl::Status ManifestIterator::Open(const std::string& manifest_file) {
     std::string msg =
         absl::StrFormat("failed to open file '%s' for reading", manifest_file);
     if (errno) {
-      status_ = ErrnoToCanonicalStatus(errno, msg);
+      status_ = ErrnoToCanonicalStatus(errno, "%s", msg);
```

Comment:
Did I mention I wished we could cross-compile?

Reply:
Knock yourself out.